source/iterators.tex

```diff
@@ -3289,15 +3294,15 @@
   template <class charT, class traits = char_traits<charT> >
   class ostreambuf_iterator {
   public:
-    typedef output_iterator_tag           iterator_category;
-    typedef void                          value_type;
-    typedef void                          difference_type;
-    typedef void                          pointer;
-    typedef void                          reference;
-    typedef charT                         char_type;
-    typedef traits                        traits_type;
-    typedef basic_streambuf<charT,traits> streambuf_type;
-    typedef basic_ostream<charT,traits>   ostream_type;
+    using iterator_category = output_iterator_tag;
+    using value_type        = void;
+    using difference_type   = void;
+    using pointer           = void;
+    using reference         = void;
+    using char_type         = charT;
+    using traits_type       = traits;
+    using streambuf_type    = basic_streambuf<charT,traits>;
+    using ostream_type = basic_ostream<charT,traits>;
 
```

Comment:
Rebased again (just to be safe) and aligned the line highlighted above - should land cleanly.